Tasks/AppCenterDistributeV3/task.json

```diff
@@ -52,6 +52,13 @@
             "required": true,
             "helpMarkDown": "Relative path from the repo root to the APK/AAB or IPA file you want to publish"
         },
+        {
+            "name": "buildVersion",
+            "type": "string",
+            "label": "Build version",
+            "required": false,
+            "helpMarkDown": "The build version of the uploading binary which needs to be specified for `.zip` and `.msi` this value will be ignored unless the platform is WPF or WinForms"
```

Comment:
Can we update: 

` this value `

to 

`. This value `

Reply:
Sure, updated 👍